beancount/core/number.py

```diff
@@ -58,21 +60,25 @@ def is_fast_decimal(decimal_module):
 
 # Constants.
 ZERO = Decimal()
-HALF = Decimal('0.5')
-ONE = Decimal('1')
+HALF = Decimal("0.5")
+ONE = Decimal("1")
 
 # A constant used to make incomplete data, e.g. missing numbers in the cost spec
 # to be filled in automatically. We define this as a class so that it appears in
 # errors that would occur from attempts to access incomplete data.
-class MISSING: pass
+class MISSING:
+    pass
+
 
 # Regular expression for parsing a number in Python.
 NUMBER_RE = r"[+-]?\s*[0-9,]*(?:\.[0-9]*)?"
 
-_CLEAN_NUMBER_RE = re.compile('[, ]')
+_CLEAN_NUMBER_RE = re.compile("[, ]")
+_CLEAN_NUMBER_RE_COMMA = re.compile("[. ]")
+
 
 # pylint: disable=invalid-name
-def D(strord=None):
+def D(strord=None, decimal_separator_comma=False):
```

Comment:
Probably a matter of taste, but personally I like if the name of an option makes it unambigiously clear that it is a boolean option. How about renaming it as follows: `use_comma_as_decimal_separator=False`?

Reply:
made the changes and updated the docstring

Reply:
Thanks! :) Btw, I should have mentioned that I am not a Beancount developer and the decision of what to name options is down to @blais, of course. I just happened to stumble across your PR and thought I'd comment. I hope I didn't accidentally step on anyone's toes in doing so.